src/fragments/lib-v1/auth/js/advanced.mdx

```diff
@@ -0,0 +1,79 @@
+## Subscribing to Events
```

Comment:
This page was taken from `lib/auth/js/advanced` (v5 docs) with the addition of Lambda triggers which were previously approved in: https://github.com/aws-amplify/docs/pull/5879

Removed all information pertaining to federation.